### DISCUSSION TITLE
Custom DKIM Text

### DISCUSSION THREAD

**Discussion by PrabhuShanmughapriyan** [postalserver/postal#2721] [Help with using Postal] [answered]
Hi,
I need to set my custom DKIM on verified domains. Is it possible to do it.. or can i get the logic how the DKIM Text is created in postal server

**Comment by willpower232** [comment, marked as answer]
Postal will be using its own key to create the DKIM so not sure you will be able to supply your own unless you mess around with the database perhaps?
